### PR TITLE
Fixed and reduced the snap size

### PR DESCRIPTION
**Pull request by soumyaDghosh**
Try this now, I have added the access to home folder. Added network access, and removed unnecessary deps. Check now if things are fine now.

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -37,10 +36,19 @@ parts:
...
+      - tooth
+    plugin: nil
+    build-snaps:  # List all content-snaps and base snaps you're using here
+      - core22
+      - gnome-42-2204
+      - gnome-42-2204-sdk
+    override-prime: |
+      set -eux
+      for snap in "<CORE-SNAP>" "<CONTENT-SNAP-1>" "<CONTENT-SNAP-2>"; do  # List all content-snaps and base snaps you're using here
+          cd "/snap/$snap/current" && find . -type f,l -exec rm -f "$SNAPCRAFT_PRIME/{}" \;
```
Comment (GeopJr) [inline snapcraft.yaml:53]: Should this be included in the store/stable release?

**Comment by GeopJr** [comment]
Thanks for this PR!

I'll keep this open until there's a resolution to https://forum.snapcraft.io/t/doesnt-use-the-xdg-desktop-portal/34360

I'd rather avoid using `home` unless there's no way to get the portal to work

**Comment by soumyaDghosh** [comment]
Keep it open, but you must make your app working until then, right?

**Comment by GeopJr** [comment]
> Keep it open, but you must make your app working until then, right?

Until I publish it to the store? Yes

**Comment by soumyaDghosh** [comment]
> Until I publish it to the store? Yes

Then you should go on with the `home` plug.

**Pull request merged**